migrate.lisp

```diff
@@ -65,10 +65,11 @@
   "Get the version of latest migration available."
   (let ((files-list (get-migration-files))
 	(counter 0))
-    (dolist (file files-list counter)
-      (unless (equal (incf counter) (get-migration-number file))
-	(warn "Migration #~S is missing!" counter)
-	(return)))))
+    (unless (eql files-list :skipped)
```

Comment:
Where does `:skipped` come from? Was it present before in the code base but not respected?